diff --git a/engine/stratagus/src/include/shaders.h b/engine/stratagus/src/include/shaders.h
index 3d4e1bd..024ddf3 100644
--- a/engine/stratagus/src/include/shaders.h
+++ b/engine/stratagus/src/include/shaders.h
@@ -3,19 +3,7 @@
 
 #include <SDL.h>
 
-#ifndef __APPLE__
 extern bool LoadShaderExtensions();
 extern bool RenderWithShader(SDL_Renderer *renderer, SDL_Window* win, SDL_Texture* backBuffer);
-#else
-#include "stratagus.h"
-
-inline bool LoadShaderExtensions() {
-    return false;
-}
-
-inline bool RenderWithShader(SDL_Renderer*, SDL_Window*, SDL_Texture*) {
-    return false;
-}
-#endif
 
 #endif
diff --git a/engine/stratagus/src/video/shaders.cpp b/engine/stratagus/src/video/shaders.cpp
index 61b52a3..9f39738 100644
--- a/engine/stratagus/src/video/shaders.cpp
+++ b/engine/stratagus/src/video/shaders.cpp
@@ -25,7 +25,6 @@
 
 #include "shaders.h"
 
-#ifndef __APPLE__
 
 #include "stratagus.h"
 
@@ -539,4 +538,3 @@ bool LoadShaderExtensions() {
 	return canUseShaders;
 }
 
-#endif
